docs/conf.py

```diff
@@ -56,9 +57,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = dg_version
+version = get_version("distgen")
 # The full version, including alpha/beta/rc tags.
-release = dg_version
+release = get_version("distgen")
```

Comment:
That's fair, but so would the previous approach, right? When it was hardcoded to the file.

Using the https://pypi.org/project/setuptools-scm/ I mentioned above might help with this, right? Also, might make the custom version generation for push/PR builds in the copr script also redundant, as if the build is made from a commit that is not being tagged with version it will be based on commit hash.

Reply:
From git-head it worked fine, if the cwd is on python's path.